Scripts/Julia/hopsan.j1

```diff
@@ -0,0 +1,53 @@
+module hopsan
+  const hopsanlibpath = joinpath(ENV["HOPSANHOME"], "bin/libhopsanc")
+  const complibpath = joinpath(ENV["HOPSANHOME"], "componentLibraries/defaultLibrary/libdefaultcomponentlibrary.so")
```

Comment:
Hard-coded .so name for library
On Windows I think the components are still built-in though, so no need to look for DLL.
But I am not sure. But on windows then we should not try to load at all.

Reply:
I added platform specific paths. It could be as you say, that we don't need to load the default library at all in Windows. I can try it, but I need to install Julia first.